infrastructure/templates/public-api/main.bicep

```diff
@@ -1,290 +1,206 @@
 
-@description('Database : administrator password')
+@description('Database : administrator password.')
 @minLength(8)
 @secure()
-param dbAdminPassword string
+param postgreSqlAdminPassword string
 
-@description('Database : Azure Database for PostgreSQL sku name ')
+@description('Database : Azure Database for PostgreSQL sku name.')
 @allowed([
   'Standard_B1ms'
   'Standard_D4ads_v5'
   'Standard_E4ads_v5'
 ])
-param dbSkuName string = 'Standard_B1ms'
-
-@description('Database : Azure Database for PostgreSQL Storage Size in GB')
-param dbStorageSizeGB int = 32
-
-@description('Database : Azure Database for PostgreSQL Autogrow setting')
-param dbAutoGrowStatus string = 'Disabled'
-
-//Container Registry Params ----------------------------------------------------------------
-@minLength(5)
-@maxLength(50)
-@description('Registry : Name of the azure container registry (must be globally unique)')
-param containerRegistryName string
+param postgreSqlSkuName string = 'Standard_B1ms'
 
-@description('Deploy the Container Registry if you are not using an existing registry')
-param deployRegistry bool
+@description('Database : Azure Database for PostgreSQL Storage Size in GB.')
+param postgreSqlStorageSizeGB int = 32
 
-//Container App Params -------------------------------------------------------------------
-@minLength(2)
-@maxLength(32)
-@description('Specifies the name of the container app.')
-param containerAppName string
+@description('Database : Azure Database for PostgreSQL Autogrow setting.')
+param postgreSqlAutoGrowStatus string = 'Disabled'
 
-@description('Specifies the name of the container app environment.')
-param containerAppEnvName string = 'publicapi'
+@description('Database : Firewall rules.')
+param postgreSqlFirewallRules array = []
 
-@description('Specifies the name of the log analytics workspace.')
-param containerAppLogAnalyticsName string = 'publicapi'
+@description('Container App : Select if you want to use a public dummy image to start the container app.')
+param useDummyImage bool = true
 
-@description('Container App : Specifies the container image to deploy from the registry <name>:<tag>.')
-param containerAppImageName string
+param now string = utcNow('u')
 
-@description('Specifies the container port.')
-param containerAppTargetPort int = 80
-
-@description('Select if you want to use a public dummy image to start the container app.')
-param useDummyImage bool
-
-
-//ServiceBus Queue Params -------------------------------------------------------------------
-@description('Name of the Service Bus namespace')
-param namespaceName string = 'processor'
-
-@description('Name of the Queue')
-param queueName string = 'Processorqueue'
+//---------------------------------------------------------------------------------------------------------------
+// Variables and created data.
+//---------------------------------------------------------------------------------------------------------------
+var project = 'ees-publicapi'
```

Comment:
Might be a good idea to make this a parameter instead? Might be useful in the future.

Of course, could just default to the current value.

Reply:
Actually I've just inlined this as it was only being used in one place.  Easy to parameterise the resourcePrefix down the line if we find a need for it, but I don;t think we have a need to right now.